alacritty/src/event.rs

```diff
@@ -197,9 +201,10 @@ impl Processor {
     /// Run the event loop.
     ///
     /// The result is exit code generate from the loop.
-    pub fn run(&mut self, event_loop: EventLoop<Event>) -> Result<(), Box<dyn Error>> {
+    pub fn run(self, event_loop: EventLoop) -> Result<(), Box<dyn Error>> {
+        let initial_window_error = self.initial_window_error.clone();
         let result = event_loop.run_app(self);
-        match self.initial_window_error.take() {
+        match initial_window_error.borrow_mut().take() {
```

Comment:
Do we ever interact with this in a way other than setting/removing the value? Seems like it uses a `RefCell` now, but maybe a `Cell` will work instead?